### DISCUSSION TITLE
Unclickable Fabricator UI Slots

### DISCUSSION THREAD

**Discussion by Antinoe** [FakeFishGames/Barotrauma#15060] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I was designing a Handheld Fabricator item when I noticed that the slots within it cannot be clicked, regardless of the items inside. Inputs can be placed in, but the crafted output cannot be removed. Items can be swapped out, however. I've provided a link for a drop-in test below:

https://github.com/Antinoe/Barotrauma-HandheldFabricator

### Reproduction steps

1. Download and enable the mod provided in the link.
2. Begin a round and grant the **handheldfabricator** item using commands.
3. Simply place items within its input slots (or the output slot) and attempt to take them out.

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by violetpuppygf** [comment]
I'm having this same problem currently, when you make any sort of handheld fabricator, make it drawhudwhenequipped so everything shows up in your hand, and craft an item, the output slot cannot be interacted with correctly. You cannot take the item out of the slot, and the only solution is to drag an item into the slot to update it.
